internal/backend/network.go

```diff
@@ -18,22 +19,33 @@ import (
 type NerdctlNetworkSvc interface {
 	FilterNetworks(filterf func(networkConfig *netutil.NetworkConfig) bool) ([]*netutil.NetworkConfig, error)
 	AddNetworkList(ctx context.Context, netconflist *libcni.NetworkConfigList, conf *libcni.RuntimeConf) (cnitypes.Result, error)
-	CreateNetwork(opts netutil.CreateOptions) (*netutil.NetworkConfig, error)
+	CreateNetwork(opts types.NetworkCreateOptions) (*netutil.NetworkConfig, error)
 	RemoveNetwork(networkConfig *netutil.NetworkConfig) error
 	InspectNetwork(ctx context.Context, networkConfig *netutil.NetworkConfig) (*dockercompat.Network, error)
 	UsedNetworkInfo(ctx context.Context) (map[string][]string, error)
 	NetconfPath() string
+	Namespace() string
 }
 
 func (w *NerdctlWrapper) FilterNetworks(filterf func(networkConfig *netutil.NetworkConfig) bool) ([]*netutil.NetworkConfig, error) {
```

Comment:
This function went [private](https://github.com/containerd/nerdctl/commit/2e3b6365cff2851b9c05b59d0ce11ff463f3357f#diff-669913e187d4dc726dcb9e01a5666ae0562549545138d25d2c8fc2ceb3e23d30R255) in nerdctl so I had to implement similar functionality in finch-daemon